netbox/dcim/models/devices.py

```diff
@@ -128,6 +129,12 @@ class DeviceType(NetBoxModel, WeightMixin):
         choices=DeviceAirflowChoices,
         blank=True
     )
+    vdc_type = models.CharField(
```

Comment:
Not sure it makes sense to tie the VDC type to the DeviceType. I see two potential scenarios:

1. The VDC type is defined by the platform, and thus not configurable, in which case there is no need to have an attribute.
2. The VDC type _is_ configurable, which would be done per instantiated device rather than on the parent type.

Reply:
I think if we go with scenario 1, the only problem is we need to have a field to define whether interface sharing is allowed, unless we just don't worry about interface sharing enforcement.

With #2, I thought about having VDC type on the VDC model, (I didn't think about on the device model), however **most** platforms only use a certain technology, as seen in the Issue and those device types are pigeon holed into that VDC technology.  I think #1 is most applicable.

Reply:
IMO we should keep things simple and go with option one, eschewing the concept of VDC type entirely until such need arises.

> the only problem is we need to have a field to define whether interface sharing is allowed, unless we just don't worry about interface sharing enforcement

For now we could ignore it, allowing an interface to be allocated to any number of VDCs, and rely on feedback from the beta to determine whether anything more robust is needed.